command.go

```diff
@@ -111,6 +111,11 @@ type Command struct {
 	// command does not define one.
 	Version string
 
+	// Data are key/value pairs of arbitrary types that can be used by applications or "plugin" libraries
+	// that wish to store and use data associated with a given command. This data will thus share the same
+	// lifetime as the command itself.
+	Data map[string]any
```

Comment:
`any` was introduced in go 1.18.
Cobra still supports go 1.15.  I know these are very old versions, but I am the opinion that if we don't need to drop support, why do it.

So, can we change `any` to `interface{}`?